packages/next/server/dev/next-dev-server.ts

```diff
@@ -700,6 +700,13 @@ export default class DevServer extends Server {
     })
   }
 
+  protected getPagesManifest(): undefined {
+    return undefined
```

Comment:
I'd maybe make it throw as it is totally unexpected to call this in the dev server.

Reply:
I refactored the base class to make the constructor always call this method, because it doesn’t know if it’s extended by the dev server or not